style.css

```diff
@@ -0,0 +1,14 @@
+/* Main text */
+.main-text {
+    float: left;

```

Comment:
@murvanessa If this should have the same width as the section from @adekimpianna and @Feruzteame maybe you can set 65%?